src/plugin/scoring-similarity/src/java/org/apache/nutch/scoring/similarity/cosine/Model.java

```diff
@@ -68,6 +68,11 @@ public static synchronized void createModel(Configuration conf) throws IOExcepti
         }
         LOG.info("Loaded custom stopwords from {}",conf.get("scoring.similarity.stopword.file"));
       }
+
+      //Check if user has specified n for ngram cosine model
+      int ngram = conf.getInt("scoring.similarity.ngrams", 1);
+      LOG.info("Value of ngram: "+ngram);
```

Comment:
Please use correct effficient slf4j code notation here
e.g. LOG.info("Value of ngram: {} ", ngram);